packages/core/upload/admin/src/pages/SettingsPage/tests/utils/server.js

```diff
@@ -4,6 +4,7 @@ import { rest } from 'msw';
 const handlers = [
   rest.get('*/settings', (req, res, ctx) => {
     return res(
+      ctx.delay(200),
```

Comment:
Why do we need to delay ?

Reply:
I throught I had to add it to make it work but it seems that I didn't needed to! :ok_hand: